instrumentation/rspec/lib/opentelemetry/instrumentation/rspec/formatter.rb

```diff
@@ -68,8 +68,12 @@ def example_started(notification)
 
         def example_finished(notification)
           pop_and_finalize_span do |span|
-            result = notification.example.execution_result
+            example = notification.example
+            result = example.execution_result
 
+            # Update name and full_description for one-liner examples where description is generated after execution
+            span.name = example.description
+            span.set_attribute('rspec.example.full_description', example.full_description.to_s)
```

Comment:
💁🏼 Use `add_attributes` as opposed to setting each attribute separately because each call requires a lock when changing a span

Reply:
Great catch. I agree. @americodls, can you update the method call to combine the attributes on line 76 and 77 into one `add_attributes` call?

Reply:
Thank you for the feedback. I just fixed that.